package.json

```diff
@@ -38,23 +32,20 @@
     "lint:commit": "ui-scripts --lint-commit",
     "lint:staged": "lint-staged",
     "update:package:list": "lerna run generate:package:list --stream --scope @instructure/instui-config",
-    "commit:package:list": "git add **/package-list.json",
+    "commit:package:list": "git add packages/instui-config/package-lists/**/package-list.json",
     "generate:component": "instui create component",
     "generate:package": "instui create package",
     "commit": "ui-scripts --commit",
-    "install:react": "ui-scripts --install-react",
-    "install:packages": "lerna bootstrap --reject-cycles",
     "bootstrap": "node scripts/bootstrap.js",
     "bootstrap:docs": "npm-run-all clean:modules install:packages clean build",
     "build": "lerna run build --stream",
     "build:watch": "lerna run build:watch --stream",
     "build:docs": "lerna run bundle --stream --scope docs-app",
-    "build:types": "lerna run build:types --stream",
```

Comment:
Too bad, I was using this one (not sure why but I prefer it over build:ts, probably because the errors are more user-friendly)

Reply:
can you please leave this? :)

Reply:
Sadly this is not really possible with yarn modern, because it takes package dependencies more seriously.
What that mean is since the sub packages (like ui-view and such) don't have `typescript` listed as their dependency, they can't call `tsc` from themselves, thus we have to call `tsc` where `typescript` is actually listed as a dependency: from the root.

However I renamed the new way of building types to the same thing as the previous one: `yarn build:types` .
This is also much better way of building them because it won't rebuild everything when you change only 1 package, it actually can determine what packages needs to be re-build and it can do it efficiently vs the old way of always rebuilding everything.